src/gbd_mapping/base_template.py

```diff
@@ -83,7 +83,8 @@ def __init__(self,
                  yll_age_group_id_start: int = None,
                  yll_age_group_id_end: int = None,
                  yld_age_group_id_start: int = None,
-                 yld_age_group_id_end: int = None, ):
+                 yld_age_group_id_end: int = None,
+                 violated_restrictions: List=[], ):
```

Comment:
Make this a tuple.  Never use mutable data structures as default args to a class initializer.